pkg/arvo/lib/hood/drum.hoon

```diff
@@ -386,7 +386,10 @@
 ::
 ++  se-adze                                           ::  update connections
   ^+  .
-  %+  roll  ~(tap in eel)
+  %+  roll
+    %+  sort
+      ~(tap in eel)
+    |=([[@ a=term] [@ b=term]] (aor a b))
```

Comment:
> always boots into `%dojo`

until we add `%zork` or other later-in-alphabet app to the default connections list. Might be worth making `%dojo` explicitly last here, so we don't run into this again.